borg/archive.py

```diff
@@ -501,7 +501,10 @@ def restore_attrs(self, path, item, symlink=False, fd=None):
             try:
                 xattr.setxattr(fd or path, k, v, follow_symlinks=False)
             except OSError as e:
-                if e.errno not in (errno.ENOTSUP, errno.EACCES):
+                if e.errno == errno.E2BIG:
+                    logger.warning('%s: Value or key of extended attribute %s is too big for this filesystem' %
+                                   (path, k.decode()))
+                elif e.errno not in (errno.ENOTSUP, errno.EACCES):
```

Comment:
this is a bit strange now: warns if we could not extract xattrs (because they were too big), but says nothing if we could not extract at all (because not supported or not permitted). this was fixed in master.